packages/backend/server/scripts/self-host-predeploy.js

```diff
@@ -1,17 +1,10 @@
 import { execSync } from 'node:child_process';
 import { generateKeyPairSync } from 'node:crypto';
 import fs from 'node:fs';
+import { homedir } from 'node:os';
 import path from 'node:path';
 
-const SELF_HOST_CONFIG_DIR = '/root/.affine/config';
-
-function generateConfigFile() {
-  const content = fs.readFileSync('./dist/config/affine.js', 'utf-8');
-  return content.replace(
-    /(^\/\/#.*$)|(^\/\/\s+TODO.*$)|("use\sstrict";?)|(^.*lint-disable.*$)/gm,
-    ''
-  );
-}
+const SELF_HOST_CONFIG_DIR = `${homedir()}/.affine/config`;
```

Comment:
For existing deployments where the container user is not root, is this considered an incompatible update? Will it cause the private key to be regenerated once, and what impact will it have?

Reply:
deployment in docker always use root user